views/pulsar/v2/helpers/html.html.twig

```diff
@@ -291,16 +291,17 @@ Show a dropdown/dropup menu when the button is clicked.
 
 ## Options
 
-Option    | Type   | Description
---------- | ------ | -----------------------------------------------------------
-caret     | bool   | Whether to show the caret arrow in the label (default true)
-class     | string | A space separated list of class names (applied to the button)
-direction | string | `down` (default) or `up`
-id        | string | A unique identifier, if required (applied to the button)
-items     | array  | An array of items to put in the dropdown list (usually links)
-label     | string | The button label
-placement | string | the alignment of the dropdown menu, `left` (default) or `right`
-data-*    | string | Data attributes, eg: `'data-foo': 'bar'` (applied to the button)
+Option       | Type   | Description
+------------ | ------ | -----------------------------------------------------------
+caret        | bool   | Whether to show the caret arrow in the label (default true)
+class        | string | A space separated list of class names (applied to the button)
+direction    | string | `down` (default) or `up`
+id           | string | A unique identifier, if required (applied to the button)
+items        | array  | An array of items to put in the dropdown list (usually links)
+label        | string | The button label
+placement    | string | the alignment of the dropdown menu, `left` (default) or `right`
+parent_class | string | A space separated list of class names (applied to the wrapper)
```

Comment:
Could you also follow the pattern used in the `form.html.twig` docs, which separates 'options applied to the parent wrapper' and 'options applied to "thing"`?